ofrak_core/ofrak/core/filesystem.py

```diff
@@ -352,13 +352,22 @@ async def initialize_from_disk(
                         ),
                     )
                 elif os.path.isfile(absolute_path):
-                    with open(absolute_path, "rb") as fh:
-                        await self.add_file(
-                            relative_path,
-                            fh.read(),
-                            file_attributes_stat,
-                            file_attributes_xattr,
-                        )
+                    try:
+                        with open(absolute_path, "rb") as fh:
+                            file_data = fh.read()
+
+                    except Exception as e:
+                        os.chmod(absolute_path, stat.S_IRUSR)
```

Comment:
Is there a reason we shouldn't just try to automatically add this permission? Right now, we're waiting for there to be an exception before trying to add the permission, but I don't see a benefit to that.